### PR TITLE
Add card

### PR DESCRIPTION
**Pull request by Walflower**
Screen Shot 2024-03-30 at 2 57 00 AM
Screen Shot 2024-03-30 at 2 57 05 AM

Created the add-card component

### CODE REVIEW

File: src/components/AddCard/AddCard.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styles from './AddCard.module.css';
+import Plus from '../../assets/images/plus.svg'
+
+
+function AddCard() {
```
Comment (FemmeQueenInSTEM) [inline src/components/AddCard/AddCard.jsx]: Perhaps this could more accurately be made `AddCardButton`? Up to you

**Review by walsquared — APPROVED** [review]

File: src/components/AddCard/AddCard.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  return (
+    <section className={styles.card}>
```
Comment (FemmeQueenInSTEM) [inline src/components/AddCard/AddCard.jsx]: Actually, this should be a react-router `Link` tag that points to "/add-application"

**Review by walsquared — CHANGES_REQUESTED** [review]

File: src/components/AddCard/AddCard.jsx (hunk 2 above)
Comment (Walflower) [reply src/components/AddCard/AddCard.jsx]: added the Link tag

File: src/components/AddCard/AddCard.jsx (hunk 1 above)
Comment (Walflower) [reply src/components/AddCard/AddCard.jsx]: yup changed

**Review by walsquared — APPROVED** [review]

**Pull request merged**